platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildScriptBuilder.java

```diff
@@ -921,6 +960,31 @@ public Type type() {
         }
     }
 
+    @NonNullApi
+    private static class StatementGroup extends AbstractStatement {
```

Comment:
💭 `AbstractStatement` and `Statement` should really be renamed to explain that they could represent multiple statements - the javadoc on `Statement` still reads "a statement".  Out of scope for this, but seeing a group of statements also be a statement is odd.